getting-started/Installation/configuration.md

```diff
@@ -29,7 +29,7 @@ To get the current amber environment just type `Amber.env.to_s`. You can also ch
 
 ### Setting the current environment
 
-Amber makes use of environment variable `AMBER_ENV` to set the current environment. By default the current environment is `development`. 
+Amber makes use of environment variable `AMBER_ENV` to set the current environment. By default, the current environment is `development`. 
```

Comment:
Amber sets the current environment based on the `AMBER_ENV` environment variable. The default environment is `development`.